doc/conf.py

```diff
@@ -190,8 +190,8 @@ def __getattr__(cls, name):
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 
-# The full version, including alpha/beta/rc tags.
-release = "0.1.0-dev"
+import catalyst
```

Comment:
@maliasadi I would just disable pylint here!
```suggestion
import catalyst  # disable=wrong-import-position
```

Reply:
In something as complicated as a config file, I think better to have the import close to where it is used.